dashboard/api_test.py

```diff
@@ -1254,15 +1254,19 @@ def test_format(self, mock_cache_refresh):
         for cache_type in CachedEdxDataApi.SUPPORTED_CACHES:
             mock_cache_refresh.assert_any_call(self.user, self.edx_client, cache_type)
 
-        assert len(result) == 2
+        assert isinstance(result, dict)
```

Comment:
Feel free to leave as is but I'm not sure what the value is for this and the next lines since we would error on `assert result['is_edx_data_fresh'] is False` anyway